### ISSUE TITLE
Question: bulk create for Companies

### ISSUE THREAD

**Issue by mkarbowiak** [intercom/intercom-ruby#277]
Is there a possibility to bulk create/update the companies?

**Comment by Skaelv** [comment]
There is no way to do that yet. I can not give you an ETA right now of when it's going to be done but it's definitely on our features list for the future.

**Comment by nickpoorman** [comment]
Would at least like the ability to bulk update companies.

**Comment by ahmed-abdulmoniem** [comment]
+1

**Comment by Startouf** [comment]
My app is currently holding a competitive event where each company's score gets recomputed every day and I would love a bulk_update capability for the Ruby API

**Comment by choran** [comment]
Hi, 
I wanted to provide an update on our bulk APIs.
We are currently reviewing our bulk APIs. As a result we have removed the Bulk APIs references from our docs. People using bulk APIs can continue to do so. We are not deprecating them. But we are encouraging newer customers to use the regular APIs for any bulk type jobs. 
We did not feel bulk APIs were offering as much benefit as we hoped so are looking at other ways of achieving this. 
In the meantime we will continue to support bulk APIs but we will not be adding them to any SDKs that do not currently have support for them. 
We will provide ongoing updates for this and other API/SDK related activity  in our [developer blog](https://developerblog.intercom.com/)
